### PR TITLE
docs(guide): 7.0.0 migration guide

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

**Comment by nduchak** [comment]
Resolve #876

File: docs/guides/migration/migration-7.0.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Migration to SDK 7.0.0
+This guide describe the process of migrating to SDK version 7.0.0
```
Comment (shekhar-shubhendu) [inline docs/guides/migration/migration-7.0.0.md]: describes

File: docs/guides/migration/migration-7.0.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+```
+#### After
+```js
+Universal({
+  accounts: [MemoryAccount({ keypair })]
+})
+```
+
+## Step 3
+Change most of `AENS` method's first argument from `nameId` to `name
```
Comment (shekhar-shubhendu) [inline docs/guides/migration/migration-7.0.0.md]: What do you mean `most of`?
Provide a list of methods for which this has changed or just say "all" if this is applicable to all

File: docs/guides/migration/migration-7.0.0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+await client.aensUpdate('testname.chain', ...)
+await client.aensTransfer('testname.chain', ...)
+await client.aensRevoke('testname.chain', ...)
+```
+
+## Other Breaking Changes
+- Add new compiler `methods` to RPC `communication` (base-app update required)
+- Drop compiler version to `version >= 4.0.0 && version < 5.0.0`
+- Change node compatibility range to `node >= 5.0.0 && node < 6.0.0`
+- Always `verify` transaction before send it to the node (can be forced used options `verify: false`)
```
Comment (shekhar-shubhendu) [inline docs/guides/migration/migration-7.0.0.md]: `transactions`

**Pull request merged**